### ISSUE TITLE
FP32 inference performance on TensorRT vs Darknet performance?

### ISSUE THREAD

**Issue by aditbhrgv** [lewes6369/TensorRT-Yolov3#38]
Hi,

Has anyone tried to compare the inference performance using Darknet model weights and converted .caffemodel weights?

I did the same for the Tiny-yolov3-3l.cfg model but I get very good performance in Darknet inference as compared to TensorRT inference code used in this repo. Ex:  In some frames, same object is detected in Darknet and not in TensorRT.
Shouldn't it be similar ? I have kept the IGNORE_THRESH = 0.25 in Yoloconfig.h and I use 0.25 conf threshold in Darknet. 
Also, NMS thresh = 0.45 in both cases.
Can anyone please let me know what can be the issues?  How can I verify the mAP or Precision, F1, Recall of my dataset in this repo? Is it achievable?

Thanks for your help !

**Comment by faedtodd** [comment]
Yes, tensorrt's performance is not good enough in yolov3-tiny, I have tried, it may caused by the process darknet2caffe.
I tried the official sample which transform the darknet weights to onnx model, and the performance is better, and fps is higher as well.
U can check it in my repo

**Comment by aditbhrgv** [comment]
@faedtodd I saw your repo. You probably used Nvidia's sample yolov3_onnx provided in TensorRT. I also used the same and the performance is much higher. 
Have you also tried to run your ONNX model in C++ TensorRT inference similar to the .caffemodel in C++ ?
I need to run ONNX model with C++ not with Python.
Thanks again !

**Comment by JensenHJS** [comment]
have you try run onnx model with C++API. Now I use python API in tensorrt-yolo. It works well. But I have to run onnx model with C++

**Comment by JensenHJS** [comment]
This is a yolov3-608.onnx model , and it's cfg file in the link. And it's original model is yolov3.weights, using python API convert it to yolov3-608.onnx,then convert onnx to yolo-608.trt. Yolov3.weights is downloaded from darknet official website. I am sorry that I don't have google cloud disk account
https://pan.baidu.com/s/19Ob69cd-fr9VP2Cziq4eXQ
